pkg/monitortests/node/legacynodemonitortests/pathological_events_test.go

```diff
@@ -108,23 +110,34 @@ func Test_testBackoffStartingFailedContainer(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			e := monitorapi.Intervals{tt.interval}
 			junits := testBackoffStartingFailedContainer(e)
+
+			// Find the junit with the namespace of openshift-etcd-operator int the testname
+			var testJunits []*junitapi.JUnitTestCase
+			for _, j := range junits {
+				if strings.Contains(j.Name, "openshift-etcd-operator") {
```

Comment:
Probably better to pass in the namespace you want to find as part of the tests struct in case there is a want / need to test for other namespaces later on

Reply:
that makes sense. thx